test/unit/lib/redmine/menu_manager/menu_helper_test.rb

```diff
@@ -75,7 +75,7 @@ def test_render_single_menu_node
     @response.body = render_single_menu_node(node, 'This is a test', node.url, false)
 
     html_node = HTML::Document.new(@response.body)
-    assert_select(html_node.root, "a.testing", "This is a test")
+    assert_select(html_node.root, "a.testing-menu-item", "This is a test")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.